packages/twenty-front/src/modules/ui/utilities/hotkey/hooks/usePreviousHotkeyScope.ts

```diff
@@ -20,6 +22,14 @@ export const usePreviousHotkeyScope = () => {
           return;
         }
 
+        if (DEBUG_HOTKEY_SCOPE === true) {
```

Comment:
do we need === true if we type the const as bool?

Reply:
This is a bug with our custom eslint rule : https://github.com/twentyhq/twenty/issues/4881